tradeexecutor/state/visualisation.py

```diff
@@ -61,7 +61,7 @@ def add_point(self,
                   value: float,
                   ):
         assert isinstance(timestamp, datetime.datetime)
-        assert isinstance(value, float), f"Got {value} ({value.__class__})"
+        assert isinstance(value, (float, type(None))), f"Got {value} ({value.__class__})"
```

Comment:
You can use `NoneType` instead of `type(None)`

https://stackoverflow.com/questions/21095654/what-is-a-nonetype-object